### PR TITLE
foss/2018a toolchain (+ HPL easyconfig as test case)

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (5 easyconfigs in this PR)
node2590.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/640fef51a41bfe589d872c88f3fdad7e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2123.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b0def119522a335c53347907ce73d1c5 for a full test report.

File: easybuild/easyconfigs/h/HPL/HPL-2.2-foss-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ arithmetic on distributed-memory computers. It can thus be regarded as a portable as well as freely available
+ implementation of the High Performance Computing Linpack Benchmark."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.netlib.org/benchmark/%(namelower)s']
+checksums = ['ac7534163a09e21a5fa763e4e16dfc119bc84043f6e6a807aba666518f8df440']
+
```
Comment (vanzod) [inline easybuild/easyconfigs/h/HPL/HPL-2.2-foss-2018a.eb]: Missing patch checksum

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.9, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.6.6
See https://gist.github.com/9f2f696479166c07fe026d3a8438542a for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
cermis - Linux debian 9.3, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/453e7716d1079a72a2e3ed1687ecd190 for a full test report.

**Comment by boegel** [comment]
Results from testing all existing easyconfigs that use the `foss/2017b` toolchain (58 at the time) with `foss/2018a`: all build without any issues whatsoever.

That's no big surprise, since the differences between `foss/2017b` and `foss/2018a` are quite minimal:

* same base (GCC 6.4.0 + binutils 2.28)
* minor OpenMPI version bump (2.1.1 -> 2.1.2)
* minor FFTW version bump (3.3.6 -> 3.3.7)
* same OpenBLAS and ScaLAPACK versions

So, this is good to go imho, to be included in EasyBuild v3.5.1.

**Pull request merged**